packages/design-system/src/components/N8nAlert/Alert.vue

```diff
@@ -202,6 +200,15 @@ const alertBoxClassNames = computed(() => {
 			}
 		}
 	}
+
+	&.background {
```

Comment:
are the changes to this component now needed?

Reply:
Not rally, maybe later, but let's reduce the noise, I'll roll it back